go.mod

```diff
@@ -6,17 +6,18 @@ require (
 	github.com/cloudflare/cfssl v1.5.0 // for `kubebuilder alpha config-gen`
 	github.com/gobuffalo/flect v0.2.3
 	github.com/joelanford/go-apidiff v0.1.0
-	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.15.0
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.17.0
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/afero v1.6.0
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/text v0.3.7 // indirect
-	golang.org/x/tools v0.1.5
-	k8s.io/apimachinery v0.22.2 // for `kubebuilder alpha config-gen`
-	sigs.k8s.io/controller-runtime v0.10.0
+	golang.org/x/tools v0.1.6-0.20210820212750-d4cc65f0b2ff
+	k8s.io/apimachinery v0.23.0 // for `kubebuilder alpha config-gen`
+	sigs.k8s.io/controller-runtime v0.11.0
 	sigs.k8s.io/controller-tools v0.7.0 // for `kubebuilder alpha config-gen`
 	sigs.k8s.io/kustomize/kyaml v0.10.21 // for `kubebuilder alpha config-gen`
-	sigs.k8s.io/yaml v1.2.0
+	sigs.k8s.io/yaml v1.3.0
 )
+
+replace k8s.io/apimachinery => k8s.io/apimachinery v0.22.2
```

Comment:
Could we remove the replace and ensure all updates into the respective scaffolds as well?

Reply:
So, the replace was done to pin the api machinary, and make it work with go 1.16 (< go 1.17). I ran into some issues before: https://github.com/kubernetes-sigs/kubebuilder/pull/2471#issuecomment-1006577690

Should I update the version of go in this PR? I will update the PR title and commit message as well, if we decide to go that route ...